zenodo/modules/spam/views.py

```diff
@@ -138,3 +219,30 @@ def safelist_add_remove(user_id):
         ))
 
     return redirect(request.form['next'])
+
+@blueprint.route('/safelist/add/bulk', methods=['POST'])
+@login_required
+def safelist_bulk_add():
+    """Add uses to the safelist in bulk."""
+    # Only admin can access this view
+    if not Permission(ActionNeed('admin-access')).can():
+        abort(403)
+    for user_id in request.post['user_ids']:
```

Comment:
```suggestion
    for user_id in request.form['user_ids[]']:
```

I think jQuery does some weird thing here with the naming... At least what I remember from other places we use jQuery :)

Reply:
This was mostly a WIP, as I didn't get a chance to test this e2e, will change if needed

Reply:
Yup, had to made that change and then some :)